File: java/NoxenCore.java
public class NoxenCore {

    public static String process(String input) {
        return "NOXEN Java Core: " + input;
    }

    public static void main(String[] args) {
        String input = args.length > 0 ? args[0] : "System Ready";
        System.out.println(process(input));
    }
}

File: cpp/noxen_core.cpp
#include <iostream>
#include <string>

std::string noxen_core(const std::string& input) {
    return "NOXEN C++ Core: " + input;
}

int main(int argc, char* argv[]) {
    std::string input = argc > 1 ? argv[1] : "System Ready";
    std::cout << noxen_core(input) << std::endl;
    return 0;
}
